src/databricks/sql/__init__.py

```diff
@@ -44,7 +44,7 @@ def TimestampFromTicks(ticks):
     return Timestamp(*time.localtime(ticks)[:6])
 
 
-def connect(server_hostname, http_path, access_token, **kwargs):
+def connect(server_hostname, http_path, **kwargs):
```

Comment:
If folks have called this function without keyword arguments in the path, e.g.
```python
connect("hostname", "path", "token")
```

This modification would break that.

Might be fine, but worth checking. @susodapop WDYT?

Reply:
thanks @pietern for catching this. will fix it shortly.

Reply:
addressed